nightwatch.conf.js

```diff
@@ -0,0 +1,17 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+const nightwatchConfig = require('terra-toolkit/lib/nightwatch/nightwatch.config.js').default;
+const webpackConfig = require('./packages/terra-site/webpack.config');
+const getPackageTestDirectories = require('terra-toolkit/lib/nightwatch/setup-helper.js').getPackageTestDirectories;
+
+let srcFolders;
+const isRepoTest = !process.cwd().includes('/packages/');
```

Comment:
For windows comp. we should update to `!process.cwd().includes('packages');`

Reply:
👍